hal/armv8m/mcx/n94x/n94x.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Phoenix-RTOS
+ *
+ * plo - operating system loader
+ *
+ * MCXN94x basic peripherals control functions
+ *
+ * Copyright 2024 Phoenix Systems
+ * Author: Aleksander Kaminski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#ifndef _HAL_MCXN94X_H_
+#define _HAL_MCXN94X_H_
+
+#include "../types.h"
```

Comment:
Should also be fine:
```suggestion
#include "types.h"
```

Reply:
There's no -I to mcx (just mcx/n94x), so I'll go with full path